### PR TITLE
Enabled bulk changes for ListView #157

### PR DESCRIPTION
**Pull request by LukasRychtecky**

### CODE REVIEW

File: is_core/generic_views/table_views.py [hunk 1]
```diff
@@ -208,18 +209,30 @@ def _get_list_filter(self):
...
+            'bulk_change_snippet_name': self.get_bulk_change_snippet_name(),
+            'bulk_change_form_url': self.get_bulk_change_form_url(),
+        })
         return context_data
 
+    def get_bulk_change_snippet_name(self):
+        return '-'.join(('default', self.model._meta.object_name.lower(), 'form'))
+
+    def get_bulk_change_form_url(self):
+        return reverse('IS:bulk-change-' + self.model._meta.object_name.lower())
```
Comment (matllubos) [inline is_core/generic_views/table_views.py]: %s or {} .format..

Comment (matllubos) [inline is_core/generic_views/table_views.py]: maybe this shoud return None if bulk change is not enabled

File: is_core/patterns.py [hunk 2]
```diff
@@ -230,7 +230,11 @@ def patterns(self):
...
         )
         result['api'] = self.pattern_class(
-            'api-%s' % self.core.get_menu_group_pattern_name(), self.core.site_name, r'^/?$', self.resource_class, self.core,
-             ('get', 'post'), clone_view_class=False
+            'api-%s' % self.core.get_menu_group_pattern_name(), self.core.site_name, r'^/?$', self.resource_class,
+            self.core, self._get_api_allowed_methods(), clone_view_class=False
         )
         return result
+
+    def _get_api_allowed_methods(self):
```
Comment (matllubos) [inline is_core/patterns.py]: this shoud be named get_api_without_id_allowed_methods (or similar)

File: is_core/generic_views/table_views.py (hunk 1 above)
Comment (matllubos) [inline is_core/generic_views/table_views.py]: where is this view defined? You must do it manually?

Comment (matllubos) [inline is_core/generic_views/table_views.py]: If it is manually you shoud define view name inside is core as parameter or method

File: is_core/rest/resource.py [hunk 3]
```diff
@@ -256,3 +259,40 @@ def _generate_form_class(self, inst, exclude=[]):
...
+        objects, errors = zip(*(self._update_obj(obj, data) for obj in qs))
+        compact_errors = tuple(err for err in errors if err)
+        return RestErrorsResponse(compact_errors) if len(compact_errors) > 0 else objects
+
+    def _update_obj(self, obj, data):
+        try:
+            return (self._create_or_update(merge(data, {self.pk_field_name: obj.pk})), None)
+        except DataInvalidException as ex:
+            return (None, self._format_message(obj, ex))
+        except (ConflictException, NotAllowedException):
```
Comment (ondrejSkowronek) [inline is_core/rest/resource.py]: why do you catch those exception if you just reraise them a dont do anything with them?

Comment (LukasRychtecky) [inline is_core/rest/resource.py]: Because I follow Piston's behavior.

**Pull request merged**